Library/MapleBacon/MapleBacon/ImageDownloadOperation.swift

```diff
@@ -20,6 +20,8 @@ public final class ImageDownloadOperation: Operation {
     fileprivate let progress: Progress = Progress()
 
     public var completionHandler: ImageDownloaderCompletion?
+    
+    public var httpAdditionalHeaders:[AnyHashable: Any]?
```

Comment:
can you fix the spaces here please: `public var httpAdditionalHeaders: [AnyHashable: Any]?`